### PR TITLE
"Officialize" renaming of examples classes to addons

### PR DESCRIPTION
**Pull request by LeviPesin**
Related issue: https://github.com/mrdoob/three.js/pull/23406

**Description**

As we now have `three/addons` alias, and started to refer to examples classes as addons, isn't it the time to note that in documentation?

### CODE REVIEW

File: docs/manual/en/introduction/Installation.html [hunk 1]
```diff
@@ -73,7 +73,8 @@ <h2>Install from CDN or static hosting</h2>
 		&lt;script type="importmap">
 		  {
 		    "imports": {
-		      "three": "https://unpkg.com/three@&lt;version&gt;/build/three.module.js"
+		      "three": "https://unpkg.com/three@&lt;version&gt;/build/three.module.js",
+                      "three/addons/": "https://unpkg.com/three@&lt;version&gt;/examples/jsm/"
```
Comment (Mugen87) [inline docs/manual/en/introduction/Installation.html:77]: Please remove this line. It is not relevant for the below minimal code example.

File: docs/manual/en/introduction/Installation.html [hunk 2]
```diff
@@ -109,21 +110,11 @@ <h2>Examples</h2>
 		</code>
 
 		<p>
-			If three.js was installed from a CDN, use the same CDN to install other components:
+			If three.js was installed from a CDN, use the same code, but with `three/addons/` in the import map.
 		</p>
 
-		<code>
```
Comment (Mugen87) [inline docs/manual/en/introduction/Installation.html:115]: Please restore this code listing with an enhanced import map that shows how to configure `three/addons/` and how to import `OrbitControls`.

File: docs/manual/en/introduction/Installation.html [hunk 3]
```diff
@@ -64,7 +64,7 @@ <h2>Install from CDN or static hosting</h2>
 
 		<p>
 			The three.js library can be used without any build system, either by uploading files to your own web server or by using an existing CDN. Because the library relies on ES modules, any script that references it must use <em>type="module"</em> as shown below.
-			It is also required to define an Import Map which resolves the bare module specifier `three`.
+			It is also required to define an import map which resolves the bare module specifier `three` (and `three/addons` for the addons, see below).
```
Comment (Mugen87) [inline docs/manual/en/introduction/Installation.html:67]: Please undo this change. `three/addons` can be mentioned further down the article.

File: docs/manual/en/introduction/Installation.html (hunk 1 above)
Comment (LeviPesin) [reply docs/manual/en/introduction/Installation.html]: Done.

File: docs/manual/en/introduction/Installation.html (hunk 2 above)
Comment (LeviPesin) [reply docs/manual/en/introduction/Installation.html:115]: Done.

File: docs/manual/en/introduction/Installation.html (hunk 3 above)
Comment (LeviPesin) [reply docs/manual/en/introduction/Installation.html]: Done.

**Review by Mugen87 — DISMISSED** [review]

File: docs/manual/en/introduction/Installation.html [hunk 4]
```diff
@@ -88,42 +88,53 @@ <h2>Install from CDN or static hosting</h2>
...
 		</p>
 
 		<code>
+		&lt;script async src="https://unpkg.com/es-module-shims@1.3.6/dist/es-module-shims.js">&lt;/script>
+
+		&lt;script type="importmap">
+		  {
+		    "imports": {
+		      "three": "https://unpkg.com/three@&lt;version&gt;/build/three.module.js",
+                      "three/addons/": "https://unpkg.com/three@&lt;version&gt;/examples/jsm/"
```
Comment (marcofugaro) [inline docs/manual/en/introduction/Installation.html:121]:
```suggestion
		    "three/addons/": "https://unpkg.com/three@&lt;version&gt;/examples/jsm/"
```

Hey @LeviPesin looks like this is wrongly formatted?

image

Comment (marcofugaro) [reply docs/manual/en/introduction/Installation.html]: still broken, I'll let you fix properly hehe 

image

**Comment by LeviPesin** [comment]
I think fundamentals.html in new manual are missing the information about import maps...

**Comment by Mugen87** [comment]
Renaming the doc pages will break a lot of URLs. Without implementing a redirect, I do not vote for doing this.

Besides, the PR got too complex. The first changes to the `Installation` guide were fine but now it's really hard to review all changes. I suggest to revert and implement changes more incrementally.

**Comment by LeviPesin** [comment]
I tried to `git revert` commits but that seems to broke everything...

**Comment by LeviPesin** [comment]
Okay, I fixed it somehow 🙏

**Comment by LeviPesin** [comment]
> Besides, the PR got too complex. The first changes to the Installation guide were fine but now it's really hard to review everything. I suggest to revert and implement changes more incrementally.

Reverted those changes.

**Comment by looeee** [comment]
@LeviPesin it might be better to open an issue for this first  to see if there is support for the change, and also to figure out exactly what is required. 

I'm personally in favor but as @Mugen87 points out it has to happen without breaking URLs. Which might make this more complicated than it first appears. On the other hand maybe it's ok just to change the docs and page titles and leave URLs as they are?

**Review by Mugen87 — APPROVED** [review]

**Comment by marcofugaro** [comment]
Hey, I believe this can be merged 👍
